granary/bluesky.py

```diff
@@ -2247,24 +2259,30 @@ def preview_delete(self, at_uri):
   def base_object(self, obj):
     return base_object(obj)
 
-  def upload_media(self, media):
+  def upload_images(self, images):
     blobs = {}
+    aspects = {}
 
-    for obj in media:
+    for obj in images:
       url = util.get_url(obj, key='stream') or util.get_url(obj)
       if not url or url in blobs:
         continue
 
       with util.requests_get(url, stream=True) as fetch:
         fetch.raise_for_status()
+        max_image_size = LEXRPC_TRUNCATE.defs['app.bsky.embed.images#image']['properties']['image']['maxSize']
```

Comment:
...actually, sorry, now that I think about it, we might want to make our own constant for this since evidently the appview doesn't actually enforce this limit. (!) https://github.com/bluesky-social/atproto/discussions/2845

Let's say 5M for now? Sorry for the runaround!

Reply:
Oh boy. Yeah no problem, done!